tests/ut_ssucli/ssuclitest.cpp

```diff
@@ -17,7 +17,7 @@
 
 typedef QStringList Args; // improve readability
 
-void SsuCliTest::init()
+void SsuCliTest::initTestCase()
```

Comment:
This doesn't seem correct. One test function may change the configuration in a way that it does not conform to the prerequisites of other test function. Why did you choose to change to init/cleanup globally?

Reply:
Because it looks just plain wrong to do the init / cleanup per testacase also based on the test asset itself.

Reply:
Unit tests are most useful when designed in a way that you can ran any subset of them in any order. Starting each test case with a clean environment is the most straightforward way to achieve that. Also notice how this choice is encouraged by QTest API authors - "init()", the short name, works per-function. So unless the initialization takes a considerable amount of time (I doubt :) ), I would prefer not changing this.

Reply:
I agree but then they should work accordingly.